client/state/account-recovery/reducer.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	ACCOUNT_RECOVERY_FETCH_SUCCESS,
+} from 'state/action-types';
+
+const reducer = ( state = {}, action ) => {
+	switch ( action.type ) {
```

Comment:
Also I see that you dispatch `ACCOUNT_RECOVERY_FETCH_FAILED` and `ACCOUNT_RECOVERY_FETCH` but they're not handled in your reducer.